src/unlimited.cpp

```diff
@@ -1336,7 +1336,7 @@ UniValue getstat(const UniValue& params, bool fHelp)
     if (params.size() < (verbose ? 4 : 3)) count = 1;  // if a count is not specified, give the latest sample
     else
       {
-	if (!params[3].isNum()) 
+	if (!params[(verbose ? 3 : 2)].isNum()) 
 	  {
```

Comment:
Ternary expressions may be considered bad, but doing integer arithmetic on booleans is arguably worse in my book...